main.cpp

```diff
@@ -243,25 +246,18 @@ Bird::~Bird()
 
 bool Bird::canSwim(bool birdFloats)
 {
-    if(birdFloats)
+    if(birdFloats && webbedFeet)
     {
-        if(webbedFeet)
-        {
-            return true;
-        }
-        return false;
+        return true;
     }
     return false;
```

Comment:
`return birdFloats && webbedFeet;` is all you need to do here.
I believe I suggested that in the previous review.
- [ ] revise

Reply:
Sorry I misunderstood that I could just return that